.readthedocs.yaml

```diff
@@ -9,6 +9,13 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  commands:
+    - python -mvirtualenv $READTHEDOCS_VIRTUALENV_PATH
```

Comment:
Was reading this PR, and noticed there's a missing space after `-m`

Also, if you want, it could be cool to add a `Makefile` recipe for this, so local users can run the same code, and CI just does `make docs`

Reply:
I was confused about the missing space myself; I copied these commands directly from the ReadTheDocs documentation and despite the missing space it works:

<img width="1015" alt="Screenshot 2023-11-23 at 9 52 38 AM" src="https://github.com/run-llama/llama_index/assets/185893/167d2b2c-f489-4ff8-ac79-0613f9d958ee">

The Makefile was in fact updated in a separate PR to include the `-j 8` thing:

https://github.com/run-llama/llama_index/blob/main/docs/Makefile#L20

On my macbook docs build ~8x faster than before!

It would probably be tidier to include these commands in the Makefile but ReadTheDocs environment was really finicky so I'm hesitant to touch it now that I've got it working :-P

Reply:
Thanks for the reply, and I don't doubt that it actually works 😸 .

To share, I just confirmed `python -m` is implemented such that it doesn't need a space after it. TIL!

```none
> python -m this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
...
Namespaces are one honking great idea -- let's do more of those!
> python -mthis
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
...
Namespaces are one honking great idea -- let's do more of those!
```

Even though `python -m module` will work without a space after `-m`, I still think it's best to have it with a space.

---

Reading this code closer now, this first line makes a virtual env using `virtualenv`, but the rest of the code never activates uses it. I am wondering if this first line using `virtual` is even necessary, I believe it's unused

---

As far as `Makefile`, I meant the one in the repo root, not `docs/Makefile`. Like there is a `watch-docs` recipe in repo root's `Makefile`, I am wondering if you can upgrade `watch-docs` (or add a new recipe) to include these changes, and then have CI just do `make watch-docs`.

Also, feel free to ignore this part, just sharing my thoughts. Having parity between local dev and CI is valuable

Reply:
Anyways, myself nitpicking a merged PR on Thanksgiving is annoying haha, so feel free to disregard these suggestions entirely. Hope you have a Happy Thanksgiving!